lib/github.ts

```diff
@@ -70,3 +70,25 @@ async function fetchIssues(args: {
   }
   return GitHubIssueResponseSchema.parse(await resp.json());
 }
+
+export async function fetchGitHubMembers(token: string, organization: string) {
+  const response = await fetch(`https://api.github.com/orgs/${organization}/members`, {
```

Comment:
I think we'll want to use [the repo contributors endpoint](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-repository-contributors) here. The org members endpoint is only going to list people who are a part of the devICT org, but we want to show everyone who has contributed even if they aren't included in the GitHub org.

It also means making a call for each individual repo. Using the same repo list that the issues list is using makes sense I think!